src/python/espressomd/cellsystem.pyx

```diff
@@ -129,6 +130,27 @@ cdef class CellSystem:
         handle_errors("Error in get_pairs()")
         return pairs
 
+    def get_neighbors(self, ParticleHandle particle, distance):
```

Comment:
```suggestion
    def get_neighbors(self, particle, distance):
```
Also don't forget `utils.check_type_or_throw_except()` on both `distance` and `particle` to make sure we pass to correct types to the C++ function.